### PR TITLE
Allows projectiles to pass portals

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

**Comment by Kromgar** [comment]
So pass through portals or just ignore them?

**Comment by CosmicScientist** [comment]
[pew pew](https://68.media.tumblr.com/062d8c4c77cf4ec4ea6db9bc57aee5ad/tumblr_oq0y99hyJB1vp3hgzo1_1280.png)

File: code/game/objects/effects/portals.dm [hunk 1]
```diff
@@ -147,7 +147,7 @@
 	var/turf/real_target = get_link_target_turf()
 	if(!istype(real_target))
 		return FALSE
-	if(!ismecha(M) && M.anchored && !allow_anchored)
+	if(!ismecha(M) && !istype(M, /obj/item/projectile) && M.anchored && !allow_anchored)
```
Comment (Iamgoofball) [inline code/game/objects/effects/portals.dm]: Why are you making it so we can't shoot lasers through portals to the other side

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/portals.dm (hunk 1 above)
Comment (ChangelingRain) [reply code/game/objects/effects/portals.dm]: goof you can't read code this is making it so you can

Comment (Iamgoofball) [reply code/game/objects/effects/portals.dm]: Fuck

**Review by Cyberboss — APPROVED** [review]

File: code/game/objects/effects/portals.dm (hunk 1 above)
Comment (Cyberboss) [reply code/game/objects/effects/portals.dm]: tbf that's what I thought too when I closed the original. The title isn't that descriptive

**Pull request merged**